lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -33,9 +33,10 @@ function multiplyFractions(node) {
   const numeratorArgs = [];
   const denominatorArgs = [];
   node.args.forEach(operand => {
-    if (Node.Type.isOperator(operand, '/')) {
-      numeratorArgs.push(operand.args[0]);
-      denominatorArgs.push(operand.args[1]);
+    if (Node.Type.isFraction(operand)) {
+      const fraction = Node.Type.getFraction(operand);
```

Comment:
Okay so I did this (https://github.com/socraticorg/mathsteps/pull/185/commits/47e5f33eb2d86c4414246dcc8000338588aaf052) , but it was a bit more complicated than expected 😂  First there was an import cycle (NodeType -> Negative -> NodeType) so I had to move some stuff out of NodeType into a new NodeHelper file. (Open to a better name if you have suggestions). And then I had to do something a little wonky with a `moveUnaryMinus` function parameter to get the unary minus stuff to work for the generalized isType/getType functions. Anyway let me know what you think, haha.